utils/generate-command-code.py

```diff
@@ -576,7 +576,7 @@ def create_command(name, desc):
 """
 /* We have fabulous commands from
  * the fantastic
- * Redis Command Table! */
+ * Server Command Table! */
```

Comment:
You need to re-generate the commands table, since you are updating this comment.